src/sql/workbench/services/resourceProvider/browser/firewallRuleDialogController.ts

```diff
@@ -49,6 +49,7 @@ export class FirewallRuleDialogController {
 		this._resourceProviderId = resourceProviderId;
 		this._firewallRuleDialog.viewModel.updateDefaultValues(ipAddress);
 		this._firewallRuleDialog.open();
+		this._firewallRuleDialog.setInitialAccountTenant(connection.azureAccount, connection.azureTenantId);
```

Comment:
Does it have to be triggered after 'open'?
It looks like can be done before, and will also trigger 'onAccountSelectionChange' event to enable/disable 'Create' button based on whether account is stale.

Reply:
Yes it can, I'll move it